### PR TITLE
added web site pages html

### PR DESCRIPTION
**Pull request by manzar79**

### CODE REVIEW

**Review by axamei — CHANGES_REQUESTED** [review]
please rename the files so they don't have spaces, e.g., `about-us.html`

**Comment by manzar79** [comment]
Removed spaces from html file names @axamei

File: About-Us.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book sale events</title>
+</head>
+<body>
+    <header>
+        <nav></nav>
+            <img src="booksale\logo.png" alt="Logo for Site">
```
Comment (axamei) [inline About-Us.html]: Please use `/` instead of `\`

File: Contact-Us.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book sale events</title>
+</head>
+<body>
+    <header>
+        <nav></nav>
+            <img src="booksale\logo.png" alt="Logo for Site">
```
Comment (axamei) [inline Contact-Us.html]: Please use `/` instead of `\`

File: Upcoming-Events.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book sale events</title>
+</head>
+<body>
+    <header>
+        <nav></nav>
+            <img src="booksale\logo.png" alt="Logo for Site">
```
Comment (axamei) [inline Upcoming-Events.html]: Please use `/` instead of `\`

File: index.html [hunk 4]
```diff
@@ -3,14 +3,29 @@
...
 <body>
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/oZlhgOBrmU8" 
-    title="YouTube video player" 
-    frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; 
-    picture-in-picture; web-share" 
-    allowfullscreen>
-    </iframe>
+    <header>
+        <nav></nav>
+            <img src="booksale\logo.png" alt="Logo for Site">
```
Comment (axamei) [inline index.html]: Please use `/` instead of `\`

**Review by axamei — CHANGES_REQUESTED** [review]
Please use `/` instead of `\`

File: About-Us.html (hunk 1 above)
Comment (manzar79) [reply About-Us.html]: updated code with /

File: Contact-Us.html (hunk 2 above)
Comment (manzar79) [reply Contact-Us.html]: updated code with /

File: Upcoming-Events.html (hunk 3 above)
Comment (manzar79) [reply Upcoming-Events.html]: updated code with /

File: index.html (hunk 4 above)
Comment (manzar79) [reply index.html]: updated code with /

**Comment by manzar79** [comment]
updated code with /

**Review by axamei — APPROVED** [review]

**Pull request merged**